php/commands/src/CLI_Command.php

```diff
@@ -129,6 +136,9 @@ public function info( $_, $assoc_args ) {
 			WP_CLI::line( "WP-CLI global config:\t" . $runner->global_config_path );
 			WP_CLI::line( "WP-CLI project config:\t" . $runner->project_config_path );
 			WP_CLI::line( "WP-CLI version:\t" . WP_CLI_VERSION );
+			WP_CLI::line( "Operating System:\t" . $system_os );
+			WP_CLI::line( "System Architecture:\t" . $system_architecture );
+			WP_CLI::line( "Current Shell:\t" . $current_shell );
```

Comment:
And this line becomes:
```
WP_CLI::line( "Shell:\t" . $shell );
```
Also could you move both lines to the start, before `PHP binary`? Seems more logical to me and looks slightly better...

(The formatting here with tabs isn't good and we should be using a table but we'll leave it for the moment...)